packages/cli/src/util/link/setup-and-link.ts

```diff
@@ -68,6 +70,15 @@ export default async function setupAndLink(
     remove(vercelDir);
   }
 
+  if (!isTTY && !autoConfirm) {
+    output.error(
+      `Command ${getCommandName(
+        'pull'
```

Comment:
I don't think you can hard code `vc pull` here because this file is also used for `vc link` and I think `vc dev`

Reply:
Ah thanks! Will fix.

Reply:
I moved the responsibility for rendering this error message out to the command that calls `setupAndLink`. This makes it clearer how to change the error message per command (and recommend the correct command argument to `autoConfirm`).

This is a good separation for this specific case, but I did not update all other cases to work this way. That something we could review and change as a whole. For now, I did export all relevant `reason` codes on error-like responses as a step towards this approach.

If there's a better way to solve this, let me know.